app/Cardano/Shell/Features/Logging.hs

```diff
@@ -46,13 +49,13 @@ data LoggingParameters = LoggingParameters
 -- the functions effects and constraining the user (programmer) of those function to use specific effects in them.
 -- https://github.com/input-output-hk/cardano-sl/blob/develop/util/src/Pos/Util/Log/LogSafe.hs
 data LoggingLayer = LoggingLayer
```

Comment:
Why not put the `m` and `a` parameters up here in the `LoggingLayer` type?

Reply:
Because maybe we don't want all our functions to have the same effects?